clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -54,6 +54,8 @@
 #include "llvm/Support/X86TargetParser.h"
 #include <sstream>
 
+#include <iostream>
```

Comment:
I don't believe this is used in your changes

Reply:
removed